app/containers/Products/InnerProductBenchmarkModal.tsx

```diff
@@ -0,0 +1,135 @@
+import React from 'react';
+import {Button, Modal, Container} from 'react-bootstrap';
+import {JSONSchema6} from 'json-schema';
+import JsonSchemaForm from 'react-jsonschema-form';
+import FormArrayFieldTemplate from 'containers/Forms/FormArrayFieldTemplate';
+import FormFieldTemplate from 'containers/Forms/FormFieldTemplate';
+import FormObjectFieldTemplate from 'containers/Forms/FormObjectFieldTemplate';
+import productSchema from './product-schema.json';
+import HeaderWidget from 'components/HeaderWidget';
+import PastBenchmarks from 'components/Benchmark/PastBenchmarks';
+import {ProductRowItemContainer_product} from 'ProductRowItemContainer_product.graphql';
+
+interface Props {
+  updateStatus: (...args: any[]) => void;
+  product: ProductRowItemContainer_product;
+  benchmarkSchema: any;
+  currentBenchmark: any;
+  editProduct: any;
+  editBenchmark: any;
+  createBenchmark: any;
+  pastBenchmarks: any;
```

Comment:
that's a lot of `any`

Reply:
haha definitely, I meant to go back and change those once I had it working